build.gradle

```diff
@@ -11,8 +11,11 @@ sourceCompatibility = JavaVersion.VERSION_1_6
 dependencies {
   compile 'io.dropwizard.metrics:metrics-core:3.1.0'
   compile 'com.newrelic.agent.java:newrelic-api:3.10.0'
-
+  compile 'com.google.guava:guava:18.0'
```

Comment:
Thats an option. When i started using metrics-nr i felt like there was not easy way of creating a custom filter, where i decide which metrics i wanted to publish. 
I agree it would be better to keep as simple as possible when talking of dependencies, so if you think it would be better to create a subproject, i´m ok. 
Would you do that so i can move this PR?